admin/tests/integration/components/users/user-detail-personal-information/authentication-method_test.js

```diff
@@ -322,6 +377,35 @@ module('Integration | Component | users | user-detail-personal-information/authe
           // then
           assert.dom(screen.queryByRole('button', { name: 'Supprimer' })).doesNotExist();
         });
+        test('it should display reassign button if it is the last authentication method', async function (assert) {
```

Comment:
Un petit saut de ligne pour le plaisir d'offrir
```suggestion

        test('it should display reassign button if it is the last authentication method', async function (assert) {
```